### ISSUE TITLE
About the train loss and validation loss < 0?

### ISSUE THREAD

**Issue by Jx-Tan** [282857341/nnFormer#40]
Hi, thanks for your working.

I found that after some epoch, the train loss and validation loss were < 0（such as train loss is -0.032, validation loss is -0.083）, what was meaning that loss < 0?

**Comment by 282857341** [comment]
The loss is the dice loss + ce loss
Generally speaking, there is a +1 in front of dice loss, but here we ignore the +1 here
